src/dialogs.cc

```diff
@@ -40,14 +40,14 @@ std::string Dialog::gtk_dialog(const boost::filesystem::path &path, const std::s
   dialog.set_position(Gtk::WindowPosition::WIN_POS_CENTER_ON_PARENT);
   
   if(title=="Save File As")
-    gtk_file_chooser_set_filename((GtkFileChooser*)dialog.gobj(), path.c_str());
+    gtk_file_chooser_set_filename((GtkFileChooser*)dialog.gobj(), path.string().c_str());
   else if(!path.empty())
```

Comment:
Think the problem was that path_c.str() returned `const wchar*` instead of `const char*` on Windows.